cg/apps/demultiplex/sample_sheet/novaseq_sample_sheet.py

```diff
@@ -124,15 +123,15 @@ def construct_sample_sheet(self) -> str:
             reagent_kit_version=self.run_parameters.reagent_kit_version,
             expected_index_length=self.run_parameters.index_length,
         )
-        sample_sheet: str = self.convert_to_sample_sheet()
+        sample_sheet_content: List[List[str]] = self.create_sample_sheet_content()
         if self.force:
             LOG.info("Skipping validation of sample sheet due to force flag")
-            return sample_sheet
+            return sample_sheet_content
         LOG.info("Validating sample sheet")
-        get_sample_sheet(
-            sample_sheet=sample_sheet,
-            flow_cell_mode="S2",
+        validate_sample_sheet(
+            sample_sheet_content=sample_sheet_content,
+            flow_cell_mode=FlowCellMode.NOVASEQ,
             bcl_converter=self.bcl_converter,
         )
         LOG.info("Sample sheet looks fine")
```

Comment:
```suggestion
        LOG.info("Sample sheet validated")
```

Reply:
Fixed in Sebastian's' PR and now part of this one